tools/picard/picard_macros.xml

```diff
@@ -81,5 +81,7 @@ This will be added shortly
 
     </token>
     
-
+    <token name="@TMP_DIR_FIX@"><!-- Bugfix that ensures that Picards temp dir is set to the galaxy temp dir -->
+        TMP_DIR=$__new_file_path__
```

Comment:
Not sure I made myself clear in the previous comments, what I meant is to use:

```
        TMP_DIR=\$TMPDIR
```
(not tested).